features/FEATURE_BLE/source/generic/GenericGap.tpp

```diff
@@ -1390,9 +1392,9 @@ void GenericGap<PalGapImpl, PalSecurityManager, ConnectionEventMonitorEventHandl
         update_random_address();
 
         // Schedule rotations every 15 minutes as recomended by the spec
-        _address_rotation_ticker.attach_us(
+        _address_rotation_ticker.attach(
             mbed::callback(this, &GenericGap::on_address_rotation_timeout),
-            15 * 60 * 1000000U
+            900s
```

Comment:
Could we use `15min` to make it clearer?

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/035fd512462acb441c3c4b2679d705c55c646d5a..f567894a294cc4e69df4e91370c2ff2ba44d9e4c).

Reply:
👍